pkg/agent/flowexporter/connections/connections.go

```diff
@@ -0,0 +1,120 @@
+package connections
+
+import (
+	"strconv"
+	"sync"
+	"time"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+var _ CxnStoreAndBuilder = new(cxnStoreAndBuilder)
+
+type CxnStoreAndBuilder interface {
+	Run(stopCh <-chan struct{})
+}
+
+// TODO: Name dilemma--cxnStoreAndBuilder or cxnCache or something else?
+type cxnStoreAndBuilder struct {
+	cxns            map[[5]string]flowexporter.Connection // Add 5-tuple as string array
+	connTrackPoller ConnTrackPoller
+	ifaceStore      interfacestore.InterfaceStore
+	mutex           sync.Mutex
+}
+
+func NewCxnStoreAndBuilder(connTrack ConnTrackPoller, ifaceStore interfacestore.InterfaceStore) *cxnStoreAndBuilder {
+	return &cxnStoreAndBuilder{
+		cxns:            make(map[[5]string]flowexporter.Connection),
+		connTrackPoller: connTrack,
+		ifaceStore:      ifaceStore,
+	}
+}
+
+func (csb *cxnStoreAndBuilder) Run(stopCh <-chan struct{}) {
+	klog.Infof("Starting conntrack polling")
+	for {
+		select {
+		case <-stopCh:
+			break
+		case <-time.After(flowexporter.PollInterval):
+			_, err := csb.poll()
+			if err != nil {
+				klog.Errorf("Error during conntrack poll cycle: %v", err)
+			}
+		}
+	}
+}
+
+// Update Connection map; If Connection is already present, update timestamp, counters etc.,
+// or add new Connection by 5-tuple of original flow along with local pod and podNameSpace.
+func (csb *cxnStoreAndBuilder) addOrUpdateCxn(cxn *flowexporter.Connection) {
+	flowTuple := [5]string{cxn.TupleOrig.SourceAddress.String(),
```

Comment:
use helper function for this after you introduce the `connectionKey` type I mentioned above:
```
func newConnectionKey(...) connectionKey {
    ...
}
```